src/reducers/url-state.js

```diff
@@ -46,6 +46,8 @@ const dataSource: Reducer<DataSource> = (state = 'none', action) => {
       return 'public';
     case 'TRIGGER_LOADING_FROM_URL':
       return 'from-url';
+    case 'PROFILE_REMOTELY_DELETED':
+      return 'unpublished';
```

Comment:
Yeah, this came from a discussion with greg. The long-term idea is to move to "unpublished" for all uses because the name "from-addon" doesn't look right anyway...